app/mailers/comment_notification_mailer.rb

```diff
@@ -1,22 +1,36 @@
 class CommentNotificationMailer < ApplicationMailer
-  def email_notification(comment)
+
+  def reviewer_notification(comment, reviewers)
     @comment = comment
     @proposal = comment.proposal
 
     # Email all reviewers of this proposal if notifications is true unless they made the comment
-    bcc = @proposal.ratings.map do |rating|
-      user = rating.user
-      if rating.event_teammate.try(:should_be_notified?) && @comment.user_id != user.id
-        user.email
-      end
+    to = reviewers.map do |reviewer|
+      reviewer.email if @comment.user_id != reviewer.id
     end.compact
 
-    if bcc.any?
+    if to.any?
       mail_markdown(
-                    bcc: bcc,
+                    to: to,
+                    from: @proposal.event.contact_email,
+                    subject: "#{@proposal.event.name} CFP: New comment on '#{@proposal.title}'")
+    end
+  end
+
+  def speaker_notification(proposal, comment, speaker_users)
+    @proposal = proposal
+    @comment = comment
+
+    to = speaker_users.map do |user|
```

Comment:
True but reviewers in the previous method isn't AR accurate either.  Perhaps change both to users to be consistent?  The mailer methods indicates one is for speakers and the other is for reviewers so that should be sufficient.